<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEditorPreferencesDialog</class>
 <widget class="QWidget" name="MapEditorPreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map editor preferences</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="movementSpeedsGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Movement speeds (cm/s, degrees/s)</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="objectRotationLbl">
      <property name="text">
       <string>Object rotation</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="objectRotationTxt">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="objectMoveLbl">
      <property name="text">
       <string>Object move</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="objectMoveTxt">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="cameraRotationLbl">
      <property name="text">
       <string>Camera rotation</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="cameraRotationTxt">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="cameraZoomLbl">
      <property name="text">
       <string>Camera zoom</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="cameraZoomTxt">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="cameraPanLbl">
      <property name="text">
       <string>Camera pan</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="cameraPanTxt"/>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>540</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="snapsGroup">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Snaps (cm, degrees)</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="gridSnapLbl">
      <property name="text">
       <string>Grid snap</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="gridSnapTxt"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="angleSnapLbl">
      <property name="text">
       <string>Angle snap</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="angleSnapTxt"/>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="useGridCheckBox">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Use grid</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="useAngleCheckBox">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Use angle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>271</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Editor environment test lightining</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Ambient light color</string>
    </property>
    <widget class="QLabel" name="redAmbientColorLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>red</string>
     </property>
    </widget>
    <widget class="QLabel" name="greenAmbientColorLbl">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>green</string>
     </property>
    </widget>
    <widget class="QLabel" name="blueAmbientColorLbl">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>blue</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="redAmbientColorTxt">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="greenAmbientColorTxt">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="blueAmbientColorTxt">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Direct light color</string>
    </property>
    <widget class="QLabel" name="redDirectColorLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>red</string>
     </property>
    </widget>
    <widget class="QLabel" name="greenDirectColorLbl">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>green</string>
     </property>
    </widget>
    <widget class="QLabel" name="blueDirectColorLbl">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>blue</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="redDirectLightColorTxt">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="greenDirectLightColorTxt">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="blueDirectLightColorTxt">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Direct light direction</string>
    </property>
    <widget class="QLabel" name="xDirectLightDirLbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QLabel" name="yDirectLightDirLbl">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>y</string>
     </property>
    </widget>
    <widget class="QLabel" name="zDirectLightDirLbl">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>z</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="xDirectLightDirTxt">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="yDirectLightDirTxt">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="zDirectLightDirTxt">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="useTestLightiningCheckBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Use test lightining</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>271</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Environement lighting</string>
   </property>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Ambient color</string>
    </property>
    <widget class="QLabel" name="redRealAmbientLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>red</string>
     </property>
    </widget>
    <widget class="QLabel" name="greenRealAmbientLbl">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>green</string>
     </property>
    </widget>
    <widget class="QLabel" name="blueRealAmbientLbl">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>blue</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="realRedAmbientTxt">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="realGreenAmbientTxt">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="realBlueAmbientTxt">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="radiusOfAddingPointLightToRenderLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius of adding light sources to the render</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="radiusOfAddingPointLightToRenderTxt">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>objectRotationTxt</tabstop>
  <tabstop>objectMoveTxt</tabstop>
  <tabstop>cameraRotationTxt</tabstop>
  <tabstop>cameraZoomTxt</tabstop>
  <tabstop>cameraPanTxt</tabstop>
  <tabstop>gridSnapTxt</tabstop>
  <tabstop>angleSnapTxt</tabstop>
  <tabstop>useGridCheckBox</tabstop>
  <tabstop>useAngleCheckBox</tabstop>
  <tabstop>redAmbientColorTxt</tabstop>
  <tabstop>greenAmbientColorTxt</tabstop>
  <tabstop>blueAmbientColorTxt</tabstop>
  <tabstop>redDirectLightColorTxt</tabstop>
  <tabstop>greenDirectLightColorTxt</tabstop>
  <tabstop>blueDirectLightColorTxt</tabstop>
  <tabstop>xDirectLightDirTxt</tabstop>
  <tabstop>yDirectLightDirTxt</tabstop>
  <tabstop>zDirectLightDirTxt</tabstop>
  <tabstop>useTestLightiningCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
